src/osd/osd_perf_counters.cc

```diff
@@ -48,13 +48,13 @@ PerfCounters *build_osd_logger(CephContext *cct) {
   osd_plb.add_time_avg(
     l_osd_op_lat,   "op_latency",
     "Latency of client operations (including queue time)",
-    "l", 9);
+    "l", PerfCountersBuilder::PRIO_UNINTERESTING);
   osd_plb.add_time_avg(
     l_osd_op_process_lat, "op_process_latency",
-    "Latency of client operations (excluding queue time)");
+    "Latency of client operations (excluding queue time)", NULL, PerfCountersBuilder::PRIO_INTERESTING);
```

Comment:
I'm missing the logic here.  The overall `op_latency` is changed to uninteresting but the partial `op_process_latency` is changed to interesting?

Reply:
This was a mistake.